joomla/joomla-cms
# Improve #10314 for readd the default behavior

Pull Request for Issue #10314
#### Summary of Changes

With 3.5 you got the items of the default menu (mainmenu) when you click the first time on submenu (on the left side after clicking "menus => manage => menu items". Now you get all menu items. This patch adds the old behavior again.
#### Testing Instructions

=> Click on the mentioned path above => all items are displayed
=> Log out => apply patch => log in => Click on the mentioned path above => the items of the first accessable menu is displayed
